### DISCUSSION TITLE
Wave Surfer - Unhandled Promise Rejection: TypeError: Load failed

### DISCUSSION THREAD

**Discussion by phonographic** [katspaugh/wavesurfer.js#3045] [Q&A]
Hi All. 
I made a WordPress plugin running wavesurfer.sj. 
Works great on all browsers but in Safari I am getting an error sometimes.
The first time I load it all is good. But if I navigate to a new page and then go back to it I get this error.

Unhandled Promise Rejection: TypeError: Load failed 

When I refresh the page a few times then it loads again. 

no problems on Firefox and on Chrome. 

here is a page link
https://midiwood.com/__aa-tester/

Any Ideas?

**Comment by phonographic** [comment]
@katspaugh Any ideas on how to fix this?

**Reply by katspaugh** [reply]
Hard to say w/o seeing the code. That it breaks on a subsequent load might indicate some caching issue.

**Reply by phonographic** [reply]
Thanks for the reply. I was streaming from dropbox whick normally works great at least on Chrome and Firefix. I have moved the files to my hosting server (Wordpress Media) and now the problem is solved. Obviously got something to do with that set up.
